website_seo/models/website.py

```diff
@@ -141,9 +141,11 @@ def get_website_view(self):
                 if '.' not in xml_id:
                     xml_id = 'website.%s' % xml_id
                 view = self.env['ir.ui.view'].search([('key', '=', xml_id)])
+                view = view and view[0] or False
```

Comment:
@kaerdsar I think you also have to update all get_website_view()[0] function calls because you return the first list element here. If you call get_website_view()[0] you would try to return the first list element again which would lead to an error.
In addition check line 137 of this file and update the view variable too if needed.